README.md

```diff
@@ -53,7 +52,7 @@ for i in range(50):
     text.insert("end", f"Line {i+1}\n")
 
 # Create the TkLineNumbers widget and pack it to the left
-linenums = TkLineNumbers(root, text)
+linenums = TkLineNumbers(root, text, justify="center", color_provider=("#2197db", "#ffffff"))
```

Comment:
IMO it doesn't make sense to call a tuple a "provider". Maybe just rename it to `colors` or something.